CHANGELOG.md

```diff
@@ -366,10 +366,10 @@ The biggest change in this release is that the Snabbdom source code has been por
 The TypeScript rewrite uses the `import` and `export` features introduced in ECMAScript 2015. Unfortunately the ES imports have no analogy to the CommonJS pattern of setting `module.exports`. This means that the Snabbdom modules that previously used this feature now have to be imported in a slightly different way.
 
 ```js
-var h = require("snabbdom/h"); // The old way
-var h = require("snabbdom/h").h; // The new way
-var h = require("snabbdom/h").default; // Alternative new way
-var { h } = require("snabbdom/h"); // Using destructuring
+h = require("snabbdom/h"); // The old way
```

Comment:
`let`/`const`?

Reply:
These changes addressed a lint error for redeclaring a variable name more than once. Removing var / let / const made that lint error go away, but another error surfaced around destructured "{ h }" and  adding "let" for that expression only `let { h } = require("snabbdom/h");` satisfied that error (sorry I don't remember details of the error.

Anyway there is a lint error if let or const are used at multiple redeclarations, so that's why "h" is redefined without those.

Reply:
Aren't there some pragma to turn off the linter for the line?

Reply:
I think so. Any way that is wanted by reviewers here is fine.
```bash
h = require("snabbdom/h"); // The old way
h = require("snabbdom/h").h; // The new way
h = require("snabbdom/h").default; // Alternative new way
let { h } = require("snabbdom/h"); // Using destructuring
```

Reply:
I'll push up a change to revert this area to its original master state, and add something to eslint-ignore the redeclaration

Reply:
looks like this now
```javascript
/* eslint-disable no-var, no-redeclare */
var h = require("snabbdom/h"); // The old way
var h = require("snabbdom/h").h; // The new way
var h = require("snabbdom/h").default; // Alternative new way
var { h } = require("snabbdom/h"); // Using destructuring
/* eslint-enable no-var, no-redeclare */
```

Reply:
what do you think @kuraga?

Reply:
Th~~~~! `var` -> `const`?

Reply:
`var` -> `const` is applied there now

Reply:
```javascript
/* eslint-disable no-redeclare */
const h = require("snabbdom/h"); // The old way
const h = require("snabbdom/h").h; // The new way
const h = require("snabbdom/h").default; // Alternative new way
const { h } = require("snabbdom/h"); // Using destructuring
/* eslint-enable no-redeclare */
```